installer_scripts/install-yb-voyager

```diff
@@ -909,8 +909,8 @@ create_guardrail_scripts_dir() {
 
 centos_install_mysql_client() {
 	version=$1
-	# If version is greater than or equal to 8 then dont install from community repo
-	if [ "$version" -ge 8 ]; then
+	# If version is equal to 8 then dont install from community repo
+	if [ "$version" -eq 8 ]; then
 		$YUM_INSTALL mysql-devel 1>&2
 		return
 	fi
```

Comment:
@ShivanshGahlot Does it makes sense to tie version here for mysql client packages in `yum` command.
like - `$YUM_INSTALL mysql-devel@version 1>&2`

Reply:
Hmm, we haven't ever faced a problem wrt the version of mysql-devel have we? I don't think its really needed I guess?

Reply:
Got it, i confused the `version` here(centos version) with mysql version